code/modules/atmospherics/machinery/components/binary_devices/passive_gate.dm

```diff
@@ -42,6 +42,10 @@
 		//Need at least 10 KPa difference to overcome friction in the mechanism
 		return 1
 
+	if(target_pressure >= input_starting_pressure - 10)
+		//Gas will not pump if the input pressure is lower than the target
+		return 1
+
 	//Calculate necessary moles to transfer using PV = nRT
 	if((air1.total_moles() > 0) && (air1.temperature>0))
```

Comment:
```suggestion
	if((air1.total_moles() > 0) && (air1.temperature > 0))
```
I know this was not apart of your PR and might be out of scope, but there are a few places where some arithmetic and comparison operators are not space properly, like the minus on line 40 and  or the arithmetic operators on line 54. Would you be willing to fix some of those?